src/components/Pages/_Department.scss

```diff
@@ -136,10 +136,17 @@
   margin-bottom: $coa-spacing-medium;
 }
 
+.coa-DepartmentPage__related-list {
+  margin-top: 20px;
+  margin-bottom: 20px;
+  margin-left: 20px;
+}
+
+/* Dept page lists style not being used, check during QA */
 .coa-DepartmentPage__related-item {
```

Comment:
are we just removing all dept page related-item styling? If so then let's get it out of here!

Reply:
just wanted to check with chase before i deleted it, removed it now!